rust.changes

```diff
@@ -16,7 +16,7 @@
 * Fri Feb 05 2021 Martin Kampas <martin.kampas@jolla.com> - 1.44.0+git10
 - [rust] Fix host-gcc use under SDK. JB#52983
 
-* Fri Jan 25 2021 Niels Breet <niels.breet@jolla.com> - 1.44.0+git9
+* Fri Jan 29 2021 Niels Breet <niels.breet@jolla.com> - 1.44.0+git9
```

Comment:
This looks like an accident.

Reply:
Add changelog entry to first line ( tomorrow :-) ).
```
* Fri Mar 15 2024 Ruben De Smet <ruben.de.smet@rubdos.be> - 1.75.0+git1
- [rust] Update to version 1.75.0. JB#61739
```

Reply:
This actually fixes the changelog since the change was merged on Friday January 29th 2021.

Reply:
> This looks like an accident.

Actually more like https://github.com/sailfishos/rust/releases/tag/1.44.0%2Bgit9

Reply:
But the actual change was committed on 29th so that is fine.